apis/pkg/v1/conditions.go

```diff
@@ -30,18 +32,52 @@ const (
 
 	// A TypeHealthy indicates whether a package is healthy.
 	TypeHealthy xpv1.ConditionType = "Healthy"
+
+	// A TypeSignatureVerificationComplete indicates whether a package's
+	// signature verification is complete. It could be either successful or
+	// skipped to be marked as complete.
+	TypeSignatureVerificationComplete xpv1.ConditionType = "SignatureVerificationComplete"
```

Comment:
My motivation was to avoid having `SignatureVerified` as `true` when verification is skipped (in most cases). But supported by a reason, it doesn't look too bad 🤔 

```
  - lastTransitionTime: "2024-10-23T16:38:51Z"
    reason: VerificationSkipped
    status: "True"
    type: SignatureVerified
```

I'll update accordingly.